src/App.css

```diff
@@ -71,7 +71,7 @@ body {
   display: flex;
   flex-direction: column;
   text-align: left;
-  max-width: 60%
+  /* max-width: 60%; */
```

Comment:
multiple max-width values make it hard to figure out how much space the element will take.
the top level max-width of 1280px is enough.